### ISSUE TITLE
Make Idle Master select games you want

### ISSUE THREAD

**Issue by BurAndBY** [JonasNilson/idle_master_extended#208]
So you can idle games you only want.

**Comment by JonasNilson** [comment]
I guess this is related to whitelist mode? Possibly a change to it?

https://github.com/JonasNilson/idle_master_extended/wiki/Whitelist-mode

**Comment by BurAndBY** [comment]
> I guess this is related to whitelist mode? Possibly a change to it?
> 
> https://github.com/JonasNilson/idle_master_extended/wiki/Whitelist-mode

Yes. I don't have that much time to find appid of 30 games. Make library option.
